### PR TITLE
Clean metrics when project is removed

### PR DESCRIPTION
**Pull request by Liuhaai**
the changes include:
1. All global metrics are moved to `./pkg/modules/metrics/metrics.go`
2. `RemoveMetrics()` is invoked when the project is deleted
3. `SRV_APPLET_MGR__MetricsCenter_Endpoint` is added into `docker-compose.yaml` to configure the endpoint of Prometheus server

### CODE REVIEW

File: cmd/srv-applet-mgr/apis/project/del.go [hunk 1]
```diff
@@ -26,6 +27,12 @@ func (r *RemoveProject) Output(ctx context.Context) (interface{}, error) {
 		return nil, err
 	}
 
+	if err := metrics.RemoveMetrics(ctx, acc.AccountID.String(), name); err != nil {
+		l := types.MustLoggerFromContext(ctx)
+		l.Error(err)
+		// return nil, err
```
Comment (Liuhaai) [inline cmd/srv-applet-mgr/apis/project/del.go:33]: the project deletion test will fail without a Prometheus server

File: cmd/srv-applet-mgr/apis/project/del.go [hunk 2]
```diff
@@ -26,6 +27,10 @@ func (r *RemoveProject) Output(ctx context.Context) (interface{}, error) {
 		return nil, err
 	}
 
+	if err := metrics.RemoveMetrics(ctx, acc.AccountID.String(), name); err != nil {
+		return nil, err
+	}
+
```
Comment (CoderZhi) [inline cmd/srv-applet-mgr/apis/project/del.go:33]: not the right place to delete, same issue for `RemoveMonitor`. e.g., if `RemoveMonitors` is successful but the `RemoveMetrics` fails, the project will still be there without monitors.
So, we may remove the return error of `RemoveMetrics` and `RemoveMonitor`, and have async checkings inside of the `RemoveMetrics` and `RemoveMonitor` to confirm the removal

File: docker-compose.yaml [hunk 3]
```diff
@@ -69,14 +70,14 @@ services:
...
     depends_on:
       - "w3bapp"
     user: root
-    command: --web.enable-lifecycle --config.file=/etc/prometheus/prometheus.yml
+    command: --web.enable-lifecycle --web.enable-admin-api --config.file=/etc/prometheus/prometheus.yml
     container_name: prometheus
     restart: always
     volumes:
-      - ${WS_WORKING_DIR:-.}/prometheus:/prometheus
+      - ./build/var/prometheus/conf/prometheus.yml:/etc/prometheus/prometheus.yml
```
Comment (CoderZhi) [inline docker-compose.yaml:81]: `./build/var/` or `${WS_WORKING_DIR}`?

Comment (Liuhaai) [reply docker-compose.yaml:81]: Currently, the only file need by the Prometheus instance is `prometheus.yml`

**Review by CoderZhi — APPROVED** [review]

**Comment by CoderZhi** [comment]
https://github.com/machinefi/w3bstream/issues/621

**Pull request merged**